TagsterWebAPI/src/TagsterWebAPI/Controllers/AdminController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Tagster.Application.Services;
+using Tagster.DataAccess.Entities;
+
+namespace TagsterWebAPI.Controllers
+{
+    [Route("admin")]
+    public class AdminController : ControllerBase
+    {
+        private readonly IAdminService _adminService;
+
+        public AdminController(IAdminService adminService)
+            => _adminService = adminService;
+
+        [HttpGet]
+        [Route("generate-fake-data/p{profilesCount}/max{maxTagsPerProfile}")]//add variables to route! (profilesCount, maxTagsPerProfile)
```

Comment:
Change code comments to documentations comment and add more details to it

Reply:
forgot to delete it, it was my "to do" kind of thing